apps/freyr/src/pages/contact-us.tsx

```diff
@@ -21,17 +22,24 @@ const ContactUsPage: FC = () => (
             href="https://app.obserfy.com/register"
             className="block mb-3 sm:mb-0 sm:mr-3"
           >
-            <Button secondary className="w-full sm:w-auto text-lg bg-green-200">Get Started</Button>
+            <Button secondary className="w-full sm:w-auto text-lg bg-green-200" onClick={() => {
+                trackCustomEvent({
+                  category: "Get Start Button",
+                  action: "Click",
+                  label: "Get Start",
+                })
+              }
+            }>Get Started</Button>
           </a>
         </div>
       </div>
       <div className="flex-row justify-center lg:w-1/3 mx-auto md:mx-32">
         <form className="bg-white shadow-md rounded p-8" name="contact" method="POST" data-netlify="true">
           <div className="py-4">
-            <input type="text" id="email" name="email" placeholder="Email" className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline" />
+            <input type="text" id="email" name="email" placeholder="Email" className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline" required={true} />
```

Comment:
I just realized this. Maybe it would be better if this has a type="email"? So that the keyboard layout would adjust to email layout and the browser will automatically validate the input as email.
```suggestion
            <input type="email" id="email" name="email" placeholder="Email" className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline" required={true} />
```